code/modules/power/apc.dm

```diff
@@ -344,7 +340,7 @@
 
 // Used in process so it doesn't update the icon too much
 /obj/machinery/power/apc/proc/queue_icon_update()
-	addtimer(CALLBACK(src, .proc/update_icon), APC_UPDATE_ICON_COOLDOWN, TIMER_UNIQUE)
+	icon_update_needed = TRUE
```

Comment:
You'll find it does if you look up 70 lines. Or literally three inches up on github since it's the change directly prior to this one